psi4/src/psi4/libmints/molecule.cc

```diff
@@ -1064,9 +1066,9 @@ void Molecule::print() const {
 
         for (int i = 0; i < natom(); ++i) {
             Vector3 geom = atoms_[i]->compute();
-            outfile->Printf("      %3s%-7s ", Z(i) ? "" : "Gh(", (symbol(i) + (Z(i) ? "" : ")")).c_str());
+            outfile->Printf("      %3s%-7s ", Z(i, true) ? "" : "Gh(", (symbol(i) + (Z(i, true) ? "" : ")")).c_str());
             for (int j = 0; j < 3; j++) outfile->Printf("  %17.12f", geom[j]);
-            outfile->Printf("  %17.12f", mass(i));
+            outfile->Printf("  %17.12f", mass(i, true));
```

Comment:
Printed masses of ghost atoms should probably be 0, but it is of little consequence.